deeplabcut/generate_training_dataset/frame_extraction.py

```diff
@@ -142,6 +143,9 @@ def extract_frames(
         The index of the camera that already has extracted frames. This will match frame numbers to extract for all other cameras.
         This parameter is necessary if you wish to use epipolar lines in the labeling toolbox. Only use if mode = 'match' and config3d is provided.
 
+    videos: list, default: None
+            A list of the string containing full paths to videos to extract frames for. If this is left as None all videos in the config file will have frames extracted.
```

Comment:
minor typo here "A list of the string containing..." --> "A list of string**s** containing..."